<launch>

	<param name="robot_description" command="$(find xacro)/xacro.py $(find pioneer3at)/urdf/pioneer3at.xacro" />
   
   
  <node pkg="robot_state_publisher" type="state_publisher" name="robot_state_publisher">
    <remap from="joint_state" to="joint_state"/>
    <param name="publish_frequency" type="double" value="15.0"/>
    <param name="tf_prefix" type="string" value=""/>
  </node>
  
  
	<node pkg="pioneer3at" type="p2os_urdf_publisher" name="p2os_urdf">
    <param name="publish_odom_test"  type="bool" value="false"/>
    <param name="publish_sonar_test" type="bool" value="false"/>
  </node>
	
</launch>
